addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -341,9 +340,18 @@
      * @return
      */
     private static int parseVolume(String vol) {
-        Matcher matcher = volumePattern.matcher(vol);
-        if (matcher.find()) {
-            return Math.round((Integer.valueOf(matcher.group(3)) + Integer.valueOf(matcher.group(7))) / 2);
+        int volumeTotal = 0, nChannels = 0;
+        for (String channel : vol.split(",")) {
```

Comment:
Bah.  The ',' in the dB for y'all was precisely why I made this patch in the first place, and then I go and straight up forget about it! :)  I will commit a change to ", " instead.